### ISSUE TITLE
Proxy target missing trying to run image rothnic/anaconda-notebook

### ISSUE THREAD

**Issue by gpillemer** [jupyter/tmpnb#147]
Hi I'm probably doing something wrong. 
I want to get rothnic/anaconda-notebook running as tmpnb. 
I have no prob getting the jupyter/minimal running. 
But getting this to work with the following command is give me a proxy target missing error. 

docker run --net=host -d -e CONFIGPROXY_AUTH_TOKEN=$TOKEN --name=tmpnb -v /var/run/docker.sock:/docker.sock jupyter/tmpnb python orchestrate.py --image='rothnic/anaconda-notebook' --container_port=8000 --command="ipython notebook --NotebookApp.base_url={base_path} --port {port}"

Could you help me out? I would really love to use this configuration with my team.

**Comment by shixudongleo** [comment]
I met the same problem, can you share your solution?

**Comment by Bhendrickx** [comment]
I encountered the same issue recently. I forgot the expose port 8888 from the container to the host. I added 'EXPOSE 8888' to the Dockerfile and rebuild the image. 
You can check whether you're facing the same problem with the command 'docker ps -a'. If the ports of the spawned containers of rothnic look like '127.0.0.1:0:8888'. If correct it should like '127.0.0.1:32768:8888'.

**Comment by ariddell** [comment]
I don't think you need to expose any ports since the ``--net=host`` option is specified. See https://docs.docker.com/articles/networking/

**Comment by Bhendrickx** [comment]
That's true for the proxy container and the tmpnb container that you're launching. But not for the spawned containers from the tmpnb (with the --image="dummy" option). If the dummy image doesn't expose the port from the ipython notebook, you'll encounter 'proxy target missing'.

**Comment by ariddell** [comment]
Oh that's right. You need it in the dockerfile.

**Comment by izahn** [comment]
Can someone give a step-by-step on how to solve this that a n00b like me can follow?
